packages/preview/src/browser/markdown/markdown-preview-handler.ts

```diff
@@ -106,7 +106,7 @@ export class MarkdownPreviewHandler implements PreviewHandler {
         if (!elementToReveal) {
             return;
         }
-        elementToReveal.scrollIntoView({ behavior: 'instant' });
+        elementToReveal.scrollIntoView();
```

Comment:
@AlexTugarev What are consequences of this change?

I wonder why it's affected. It's not part of Node.js, but DOM api.

Reply:
@akosyakov, `instant` scroll behavior was experimental, apparently they removed it leaving `smooth` and `auto`. this change shouldn't make any difference. 

fyi, `instant` was in place just to make sure it isn't using `smooth` by default, which would be  bogus as it's not an atomic operation.